### ISSUE TITLE
Crash when viewing certain profiles 

### ISSUE THREAD

**Issue by Suzie97** [mastodon/mastodon-ios#440]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The app crashes while trying to view some profiles. The posts, following and followers count all show as 999.

### Expected Behavior

The profile should be normally viewable like other profiles that work.

### Steps To Reproduce

1. Visit this profile : @hazelnot@sunbeam.city
2. The app crashes showing a broken profile screen.

### Environment

```markdown
- Device: iPhone 12 mini
- OS: iOS 
- Version: 15.5
- Build:
```

### Anything else?

This happens with a few select profiles other than the one mentioned in the reproduction steps above. I can’t really figure out any relation between the profiles that are a subject to this issue.

**Comment by Suzie97** [comment]
Here is a screenshot of the bug: 
![image](https://user-images.githubusercontent.com/68198116/170122486-2c1dfc81-d674-4ebf-b4c0-d41fd57b9f39.png)

**Comment by MainasuK** [comment]
I just try to visit this profile and cannot reproduce this issue. Maybe some post is not public and followers only visible. Please tell me if there is another profile without following could reproduce this problem. It's looks like some posts will crash the app likes #212. And #394 may related.
 
Thank you for your feedback.

**Comment by Suzie97** [comment]
> Please tell me if there is another profile without following could reproduce this problem.

I finally found another profile that crashes 

![image](https://user-images.githubusercontent.com/68198116/171339523-a416902e-4f84-4fe4-809c-b36c935bdc1d.png)

**Comment by MainasuK** [comment]
It should be fixed in v1.4.3. Please tell me if the issue still exists. :)
